clusterloader2/pkg/util/cluster.go

```diff
@@ -88,11 +100,13 @@ func isNodeSchedulable(node *corev1.Node) bool {
 	return !node.Spec.Unschedulable && nodeReady && networkReady
 }
 
-// Tests whether node doesn't have any taint with "NoSchedule" or "NoExecute" effect.
+// Tests whether node doesn't have any built-in taint.
 func isNodeUntainted(node *corev1.Node) bool {
-	for i := range node.Spec.Taints {
-		if node.Spec.Taints[i].Effect == corev1.TaintEffectNoSchedule || node.Spec.Taints[i].Effect == corev1.TaintEffectNoExecute {
```

Comment:
Let's keep NoSchedule/NoExecute checks.
PreferNoSchedule / PreferNoExecute should not block nodes from being counted as it could be useful somehow for test setup.

Reply:
Done.